### DISCUSSION TITLE
Is there a gesture to shut down the device?

### DISCUSSION THREAD

**Discussion by mcoy001** [microsoft/MixedRealityToolkit-Unity#10148] [General]
![image](https://user-images.githubusercontent.com/47279367/128770409-1459581b-1adb-4510-947f-c5d34695ed1f.png)

Hi, I'm working on my own project, in a mini game that I use left hand to select weapon type and right hand to do trigger pose to fire.
It works, but in very rare case the device would pop up a line asking me if I want shut down the device. I wasn't saying a word I guess maybe I triggered some universal gesture as shut down command, or it could be my imagination. I don't find any related in the basic instruct ruction page, I'm just curious if anyone know there's such thing?(or some MRTK feature)

**Comment by RogPodge** [comment]
Hey there, to our knowledge there's no such universal gesture to shut down the device. We do know that there is a voice command to initiate a shut down, but since you weren't saying anything that couldn't have been the case.

Do you have a screenshot of the shutdown prompt? This might help us determine what's causing it.

**Reply by mcoy001** [reply]
Thanks for replying, this is what I ran when I had the shut down pop up, https://vimeo.com/manage/videos/585552428
but I can't reproduce the shut down pop. I remember the battery was about 30% and my room is quite empty so maybe it was some echo or noise from outside being picked up by the device. I will update if I have it again.
